<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="solve">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>361</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Find the best cuts</string>
    </property>
   </widget>
   <widget class="QListWidget" name="eBarList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eBarLength">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eBarNumber">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="solution">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="eAdd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bar length</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of bars</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Existing Bars</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>The Solution presented as profiles to be cut </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of bars</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bar length</string>
    </property>
   </widget>
   <widget class="QListWidget" name="rBarList">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="rBarNumber">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rAdd">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rBarLength">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Requested Bars</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
